packages/sanity/src/tasks/src/tasks/components/form/addonWorkspace/taskSchema.tsx

```diff
@@ -17,7 +17,7 @@ const targetContentField = (mode: FormMode) =>
   defineField({
     type: 'object',
     name: 'target',
-    title: 'Target content',
+    title: 'Target',
```

Comment:
Noticing this, we will need to find a way later to provide translations to this I guess... 
It should be possible to send the `t` function from https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/tasks/src/tasks/components/form/addonWorkspace/TasksAddOnWorkspaceProvider.tsx#L43-L44

Reply:
yeah, I'll take another pass in a follow up PR